### PR TITLE
Log4j most commonly shared indicators (Dec 10-13 2022)

### PR DESCRIPTION
**Pull request by gio-pecora**
Guidance required for adding the following hosted YARA rule into the template: https://github.com/Neo23x0/signature-base/blob/master/yara/expl_log4j_cve_2021_44228.yar

### CODE REVIEW

File: Emerging_Threats/log4j.yml [hunk 1]
```diff
@@ -0,0 +1,1151 @@
...
+rules:
+  'Log4j #001':
+    namespace: general
+    detect:
+      events:
+      - NETWORK_CONNECTIONS
+      op: or
+      rules:
+      - op: is
+        path: event/NETWORK_ACTIVITY/?/DESTINATION/IP_ADDRESS
```
Comment (maximelb) [inline Emerging_Threats/log4j.yml]: Do we only care about Destination?

I think what you want is this:
```suggestion
        path: event/NETWORK_ACTIVITY/?/IP_ADDRESS
```

File: Emerging_Threats/log4j.yml [hunk 2]
```diff
@@ -0,0 +1,1151 @@
...
+        value: 212.47.237.67
+    respond:
+    - action: report
+      name: log4j_2021-12-13
+  'Log4j #002':
+    namespace: general
+    detect:
+      events:
+      - ONGOING_IDENTITY
```
Comment (maximelb) [inline Emerging_Threats/log4j.yml:1097]:
```suggestion
      - ONGOING_IDENTITY
      - CODE_IDENTITY
```

**Review by maximelb — CHANGES_REQUESTED** [review]

**Review by gio-pecora — COMMENTED** [review]
Approved

**Comment by gio-pecora** [comment]
https://github.com/refractionPOINT/rules/blob/log4j/Emerging_Threats/log4j.yml

**Review by gio-pecora — COMMENTED** [review]
Template optimised as requested

**Comment by gio-pecora** [comment]
Closed by mistake

**Review by maximelb — APPROVED** [review]

**Pull request merged**